website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -382,6 +384,22 @@ The `identity` block exports the following:
 
 ---
 
+The `identity_profile` block exports the following:
+
+* `kubelet_identity` - A `kubelet_identity` block as defined below.
+
+---
+
+The `kubelet_identity` block exports the following:
+
+* `client_id` - The client id of the user-defined Managed Identity assigned to the kubelets.
+
+* `object_id` - The object id of the user-defined Managed Identity assigned to the kubelets.
+
+* `resource_id` - The resource id of the user-defined Managed Identity assigned to the kubelets.
```

Comment:
we avoid the phrase `Resource ID` since it's ambiguous between Terraform and Azure Resource ID's - could we instead make this `user_assigned_identity_id`

```suggestion
* `user_assigned_identity_id` - The ID of the User Assigned Identity assigned to the Kubelets.
```